### ISSUE TITLE
[BUG] gopath/pkg/mod/github.com/asim/go-micro/plugins/server/grpc/v3@v3.0.0-20210924081004-8c39b1e1204d/grpc.go:401:6: undefined: logger.Extract

### ISSUE THREAD

**Issue by fanaticism** [go-micro/go-micro#2288]
`gopath/pkg/mod/github.com/asim/go-micro/plugins/server/grpc/v3@v3.0.0-20210924081004-8c39b1e1204d/grpc.go:401:6: undefined: logger.Extract`

Is this a bug?
Unable to run go-micro

**Comment by AuditeMarlow** [comment]
Can you provide a reproducible example?

**Comment by gregkv** [comment]
From PR #2278 grpc server plugin started to use new Extract function from logger package. This function is not available in go-micro v.3.6.0 (last release from 23 Aug).

@fanaticism check your go.mod file - seems like you need to import latest go-micro/v3 from master (or keep go-micro/v3 v.3.6.0 and import old grpc server plugin). 

Here is an example of working "pair" with latest changes:
```
github.com/asim/go-micro/plugins/server/grpc/v3 v3.0.0-20210929061216-4612baa7f814
github.com/asim/go-micro/v3 v3.6.1-0.20210929061216-4612baa7f814
```

And if you would like to keep go-micro v3.6.0:
```
github.com/asim/go-micro/plugins/server/grpc/v3 v3.0.0-20210812172626-c7195aae9817
github.com/asim/go-micro/v3 v3.6.0
```

**Comment by fanaticism** [comment]
Thanks, but why not release an updated tag and make sure the code after ` go get github.com/asim/go-micro/v3` is available. @gregkv  Excuse my poor English

**Comment by xpunch** [comment]
go-micro v3 and plugins were released with 3.7.0, now you can upgrade your go-micro and plugins to 3.7.0.
```
go get github.com/asim/go-micro/v3@v3.7.0
go get github.com/asim/go-micro/plugins/server/grpc/v3@v3.7.0
```

**Comment by gregkv** [comment]
@xpunch Thanks for the update! I believe this bug can be closed now
